packages/api/tsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "extends": "@acme/tsconfig/base.json",
   "compilerOptions": {
-    "tsBuildInfoFile": "node_modules/.cache/tsbuildinfo.json"
+    "tsBuildInfoFile": "node_modules/.cache/tsbuildinfo.json",
+    "paths": {
```

Comment:
relative path is not working, with tsconfig. It's already configured through package.json of API folder.

use leader <@acme/validators>